docs/Add-a-module.md

```diff
@@ -0,0 +1,63 @@
+
+## **How to add a module**
+
+
+1. Get the Module From this website [http://www.azerothcore.org/modules-catalogue/](http://www.azerothcore.org/modules-catalogue/)
+2. Clone it, or download.
+3. I'll use this for Exmple: [http://www.azerothcore.org/modules-catalogue/details.html?id=177019524](http://www.azerothcore.org/modules-catalogue/details.html?id=177019524).
+4. Once you downloaded it or cloned: ( Extract it, you will have than Folder looks like this :
+[[images/1.png]]
```

Comment:
the images don't seem to work: https://github.com/azerothcore/wiki/blob/13e6ec8c3dcb6e784d33b62d7e599ca98fabfca5/docs/Add-a-module.md